brut.apktool/apktool-lib/src/test/resources/brut/apktool/aapt2/testapp/apktool.yml

```diff
@@ -0,0 +1,18 @@
+version: 2.0.0
```

Comment:
Lets remove this entire testapp and start it over with AAPT2 in mind. Most of our test suite is isolated to AAPT1. It doesn't make sense to force a test suite that is filled with AAPT1 limitations/tweaks into AAPT2. As AAPT2 issues/regressions are discovered, we can add them to the aapt2 test suite.

Reply:
Rewrote the test suite. Dropped all aapt1 tests in aapt2.